qsirecon/config.py

```diff
@@ -567,83 +541,14 @@ def _process_value(value):
 
-    anat_modality = None
-    """Modality to use as the anatomical reference. Images of this
-    contrast will be skull stripped and segmented for use in the
-    visual reports and reconstruction. If --infant, T2w is forced."""
-    anat_only = False
-    """Execute the anatomical postprocessing only."""
-    anatomical_template = None
-    """Keeps the :py:class:`~niworkflows.utils.spaces.SpatialReferences`
-    instance keeping standard and nonstandard spaces."""
-    b0_threshold = None
-    """Any value in the .bval file less than this will be considered a b=0 image."""
-    b0_motion_corr_to = None
-    """Perform SHORELine's initial b=0-based registration to first volume?
-    Or make a template? Either 'iterative' or 'first'"""
-    b0_to_t1w_transform = None
-    """Transformation model for intramodal registration."""
-    b1_biascorrect_stage = None
-    """The stage of processing at which to apply B1 bias correction. Either "final" (after
-    resampling), "none" (skipped entirely) or "legacy" (before concatenation)."""
-    cifti_output = None
-    """Generate HCP Grayordinates, accepts either ``'91k'`` (default) or ``'170k'``."""
-    denoise_after_combining = False
-    """Run ``dwidenoise`` after combining dwis, but before motion correction."""
-    denoise_method = None
-    """Image-based denoising method. Either "dwidenoise" (MRtrix), "patch2self" (DIPY)
-    or "none"."""
-    distortion_group_merge = None
-    """How to combine images across distortion groups (concatenate, average or none)."""
-    do_reconall = True
-    """Run FreeSurfer's surface reconstruction (ignored)."""
-    dwi_denoise_window = None
-    """Window size in voxels for image-based denoising, integer or "auto"."""
-    dwi_no_biascorr = None
-    """DEPRECATED: see --b1-biascorrect-stage."""
-    dwi_only = False
-    """DEPRECATED: True if anat_modality is 'none'."""
-    eddy_config = None
-    """Configuration for running Eddy."""
-    fmap_bspline = None
-    """Regularize fieldmaps with a field of B-Spline basis."""
-    fmap_demean = None
-    """Remove the mean from fieldmaps."""
-    force_syn = None
-    """Run *fieldmap-less* susceptibility-derived distortions estimation."""
-    hmc_model = None
-    """Model used to generate target images for hmc."""
-    hmc_transform = None
-    """Transformation to be used in SHORELine."""
-    ignore = None
-    """Ignore particular steps for *QSIRecon*."""
     infant = False
     """Configure pipelines specifically for infant brains"""
-    intramodal_template_iters = None
-    """Number of iterations for intramodal template construction."""
-    intramodal_template_transform = None
-    """Transformation used for building the intramodal template."""
     longitudinal = False
     """Run FreeSurfer ``recon-all`` with the ``-logitudinal`` flag."""
-    no_b0_harmonization = False
-    """Skip re-scaling dwi scans to have matching b=0 intensities."""
-    output_resolution = None
```

Comment:
I think ScalarMapper still needs this

Reply:
It doesn't seem to be used in #11

Reply:
I see now that it's because I removed `init_output_grid_wf`, but I shouldn't have. I've added it back in.